src/FocusRing.tsx

```diff
@@ -77,6 +77,12 @@ export default function FocusRing(props: FocusRingComponentProps) {
     ringContext.invalidate();
   });
 
+  // If this ring no longer is enabled, hide it.
+  React.useEffect(() => {
+    if (!enabled) ringContext.hide();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [enabled]);
```

Comment:
I think this should include `ringContext`, and probably doesn't need the eslint disable. (I don't think the one below it needs the disable either, but is a holdover from a previous refactor).

`ringContext` could technically change if a component gets remounted somewhere else in a tree, granted this is pretty unlikely but being safe is always good.

Reply:
Resolved below, thanks.